external/uap/uap.depproj

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <!-- This project restores and publishes the targeting pack for specific versions of UAP.
+       With this, we are able to compile assemblies against shipped, stable versions of UAP. -->
+  <PropertyGroup>
+    <BinPlaceRef>true</BinPlaceRef>
+    <NuGetDeploySourceItem>Reference</NuGetDeploySourceItem>
+    <UAPPackageVersion>6.0.7</UAPPackageVersion>
+    <TargetFramework>uap10.0.16299</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform">
+      <Version>$(UAPPackageVersion)</Version>
+    </PackageReference>
+    <!-- System.Reflection.Emit and System.Reflection.Emit.ILGeneration are needed for the closure of the implementation of System.Reflection.DispatchProxy.
```

Comment:
I don't think you need Ref.Emit nor ILGeneration for your projects, so you should be able to remove everything from lines 17-28.